src/index.ts

```diff
@@ -9,17 +9,19 @@ const handlers: { [key: string]: Handler } = {};
 const myName = process.env.name;
 const myPmId = Number(process.env.pm_id);
 let myBus: any;
+const globalResponseListeners: Record<string, any> = {};
 
-process.on('message', async ({ topic, data: { targetInstanceId, data } }: RequestPacket): Promise<void> => {
-  if (typeof handlers[topic] === 'function' && process.send) {
-    const response: ResponsePacket<any> = {
-      type: `process:${targetInstanceId}`,
-      data: { instanceId: myPmId, message: await handlers[topic](data) },
-    };
+process.on('message', async ({ topic, data: { targetInstanceId, data, requestId } }: RequestPacket): Promise<void> => {
```

Comment:
new random guid `requestId` is added to each request for metrics to distinguish concurrent calls to the same process